SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -596,6 +646,19 @@ Method : FunctionDef {
 		^[names, values].flop.flatten
 	}
 
+	isClassMethod {	^ownerClass.isMetaClass	}
+
+
+	makePerformableEnvir {|argsArray=([]), keywordArgsEvent=(())|
```

Comment:
I'd shift the default arguments to the *last* possible moment (lazy style). They are defined already in `prMakePerformableEnvir`.